From 01a66a7d14f3e0ba328d8e0dd2dd597762bc5698 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Wed, 5 May 2021 13:38:00 +0200
Subject: [PATCH] Stop forcing static builds

When not hardcoding static, the library can be build with shared libs or
without, depending on what value the user passes to -DBUILD_SHARED_LIBS
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3da0c8d..cff4da2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,7 @@ else()
     list(APPEND library-src src/gauss_transform_make_default.cpp)
 endif()
 
-add_library(Library-C++ STATIC ${library-src})
+add_library(Library-C++ ${library-src})
 set_target_properties(Library-C++ PROPERTIES
     OUTPUT_NAME cpd
     VERSION ${CPD_VERSION}
-- 
2.31.1

